PowerShell/JumpCloud Commands Gallery/Linux Commands/Linux - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,45 @@
+#### Name
+
+Linux - Set JumpCloud Password Manager's Release Channel | v1.0 JCCG
+
+#### commandType
+
+linux
+
+#### Command
+
+```
+#!/bin/bash
+
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public ON LINE 4 depending on your desired release channel
```

Comment:
LINE 4? I think we should just say change the variable value. If you copy and paste this today, it's LINE 5